Camera research (#6)
maxlintelo/speed-camera: Documentation/RD_SpeedCamera/content/camera.tex

\chapter{Camera}
\label{chap:camera}

> What do camera’s need to read number plates from vehicles?

Reviewer: ```suggestion
What do cameras need to read number plates from vehicles?
```

\begin{itemize}
    \item High resolution: A high-resolution camera can capture detailed images, which is important for accurately capturing the numbers and letters on a number plate.
    \item Image stabilisation: Image stabilisation is needed to ensure that the number plate is clearly visible even if the vehicle moves quickly.
    \item Special lenses: To capture the number plate clearly, the camera needs special lenses designed to focus on a limited area and minimise distortion.
    \item Infrared illumination: Many number plate cameras use infrared illumination to clearly illuminate the number plate, even in low-light situations.
    \item Image processing technology: To recognise the licence plate, the camera needs advanced image processing technology, such as optical character recognition (OCR), to identify and interpret the numbers and letters on the licence plate.
\end{itemize}

> \textbf{Recognizing a car driving down the road requires a camera. Available cameras are compared on the following points:}

Reviewer 2: waarom geen section ipv bold?

\begin{enumerate}
    \item \textbf{Frame rate}\\

>     The number of frames a camera records per second is called the frame rate. The frame rate determines how smoothly you play back the images. You use a high frame rate to film fast movements smoothly. However, be aware that images can quickly become underexposed because there is only a short amount of light falling on the sensor during filming. So more images of the car will give you more information, but if you position the camera correctly, you will only need 1 image to read the license plate properly. If the car drives straight at the camera, 1 image per second is basically enough to read the license plate.

Reviewer: Reference figure in tekst

>     \begin{linfigure}{Example of fps.}

Reviewer 2: geef elk figure een tag
```suggestion
    \begin{linfigure}{fig:example-fps}{Example of fps.}
```

        \includegraphics[width=0.35\textwidth]{image_fps}
    \end{linfigure}

    \item \textbf{Field of view}\\

>     The area across which your camera can image is known as the field of view or FOV, the larger the FOV the more of your sample you can see. Having a large FOV allows you to take more efficient images containing more data, and take fewer images in order to capture the entire sample. So you are tempted to say the FOV of the camera/lens needs to be large enough so that the whole road can be seen in width. Nevertheless if there are too many pixels in 1 frame, then its possible that the car will get blurry and you won’t be able to read the license plate of the car. So to read a license plate properly then, the car has to be close enough. For example the photo will look like this if the FOV are too high:

Reviewer: Reference figure in tekst

>     \begin{linfigure}{Example of image taken with high FOV.}

Reviewer 2: geef elk figure een tag

        \includegraphics[width=0.35\textwidth]{example_high_fov}
    \end{linfigure}

> \newpage

Reviewer 2: waarom
```suggestion
```

    The license plate is totally unreadable now. So a smaller FOV will have to be used, it will make a sharper image from a smaller area. The following example uses a camera at slightly greater distance but has a smaller FOV. More pixels are pointed on the car so it is sharper. The camera in this example has a 62-degree field of view and a 12mm lens:

>     \begin{linfigure}{Example of image taken with an average FOV.}

Reviewer 2: geef elk figure een tag

        \includegraphics[width=0.35\textwidth]{example_mid_fov}
    \end{linfigure}
    \\The license plate of the car is now reasonably visible but letters/numbers like Z and 2 and I and 1 are still difficult to distinguish. To counteract this, an even smaller FOV will have to be chosen so that even more pixels can be focused on the car.

>     The following example shows an image of a car at a distance of about 50 meters filmed with an FOV of 4 degrees with a 60 mm lens. 

Reviewer: Reference figure in tekst

>     \begin{linfigure}{Example of image taken with a low FOV.}

Reviewer 2: geef elk figure een tag

    \\
    \item \textbf{Power Usage}\\
    The whole system must be able to be powered via a power bank. Ideally, the camera should have enough power in the range of 3-5 volts.

    In conclusion, the camera does not need a high FPS to avoid overexposure. The camera needs to be positioned in the right way, the angle should be almost straight in front of the cars so that the smallest possible FOV can be chosen to make a sharper images of the cars. In addition, a fast shutter speed is essential to capture images of a license plate at night as well. And finally, the system must be able to be powered by a power bank, or in other words with a low voltage of 3-5V.
\end{enumerate}

\newpage

> \textbf{The different interfaces to communicate with the camera module:}

Reviewer 2: waarom hier ook geen section?

        \lintablerow{ Bandwidth      & 320 MB/s per lane    & 400 MB/s}
        \lintablerow{ Cable length   & <30 cm               & <5 m}
        \lintablerow{ Plug and play  & No                   & Yes}
    \end{lintabular}
\end{lintable}

So the conclusion is that MIPI CSI-2 is faster than USB3.0. In addition, MIPI requires less from the CPU due to its multicore processors. This leaves the CPU more space for application tasks like vision, leading to better overall performance of the system. In theory, the maximum bandwidth of the USB interface is 5 gigabits per second. Practically, you could attain a bandwidth of about 3.6 gigabits per second. This limits the ability of vision systems used in many applications to transfer image data fast for various processing and analysis purposes. That is where CSI-2 can be more helpful. It offers a maximum bandwidth of 6 gigabits per second, with an attainable bandwidth of roughly 5 gigabits per second). USB is plug and play which has the advantage that if the camera is broken it can be replaced for another one. Across the board, MIPI CSI-2 has the most advantages and is the interface of choice.

> \newpage

Reviewer 2: kan weg?

> \textbf{MIPI CSI 2}\\

Reviewer 2: ```suggestion
\textbf{MIPI CSI 2}

```

> \begin{linfigure}{Example of the mipi csi-2 interface.}

Reviewer 2: geef elk figure een tag

> \begin{linfigure}{Example of the mipi csi-2 interface.}

Reviewer: Reference figure in tekst

    \includegraphics[width=0.60\textwidth]{mipicsi_2}

> \end{linfigure}

Reviewer 2: ```suggestion
\end{linfigure}

```
